tox.ini

```diff
@@ -40,7 +40,7 @@ commands = flake8 .
 [testenv:docs]
 basepython = python3
 usedevelop = false
-whitelist_externals =
+allowlist_externals =
```

Comment:
As this is new in 3.18, I think we should add the `minversion` configuration:

https://tox.readthedocs.io/en/latest/config.html#conf-minversion

Reply:
:+1: